pkg/rke/k8s_rke_system_images.go

```diff
@@ -9637,6 +9678,47 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
 
+		"v1.26.11-rancher2-2": {
```

Comment:
Add comment to note when this version is planned to be introduced

Plus, for consistency remove newline between these 1.26 versions but keep it before
```
		// k8s version from 2.1.x release with old rke-tools to allow upgrade from 2.1.x clusters
		// without all clusters being restarted
		"v1.11.9-rancher1-3": {
```

```suggestion
		// Enabled out of band post v2.7.9
		"v1.26.11-rancher2-2": {
```

Reply:
Done.

Reply:
Thank you. I know it's extremely nitpicky, but empty line before
```
		// k8s version from 2.1.x release with old rke-tools to allow upgrade from 2.1.x clusters
		// without all clusters being restarted
		"v1.11.9-rancher1-3": {
```
got removed (I mean https://github.com/rancher/kontainer-driver-metadata/blob/169f26600f6512876a0102bf558a3ebd90d52949/pkg/rke/k8s_rke_system_images.go#L9639). Please restore that empty line just in case and for consistency.